readthedocs/templates/projects/notifications/deprecated_build_image_used_email.txt

```diff
@@ -0,0 +1,31 @@
+{% extends "core/email/common.txt" %}
+{% block content %}
+The Read the Docs build system is deprecating "build.image" config key on ".readthedocs.yaml" starting on October 16, 2023.
+We are sending weekly notifications about this issue to all impacted users,
+as well as temporary build failures (brownouts) as the date approaches for those who haven't migrated their projects.
+
+The timeline for this deprecation is as follows:
+
+* Monday, August 28, 2023: Do the first brownout (temporarily enforce this deprecation) for 12 hours: 00:01 PST to 11:59 PST (noon)
+* Monday, September 18, 2023: Do a second brownout (temporarily enforce this deprecation) for 24 hours: 00:01 PST to 23:59 PST (midnight)
+* Monday, October 2, 2023: Do a third and final brownout (temporarily enforce this deprecation) for 48 hours: 00:01 PST to October 3, 2023 23:59 PST (midnight)
+* Monday, October 16, 2023: Fully remove support for building documentation using "build.image" on the configuration file
+
+We have identified that the following projects which you maintain, and were built in the last year, are impacted by this deprecation:
```

Comment:
This says last year, so the above date should be 2022 ^^

Reply:
```suggestion
We have identified that the following projects which you maintain, and were built in the last 3 years, are impacted by this deprecation:
```